articles/purview/register-scan-azure-sql-database.md

```diff
@@ -154,8 +154,9 @@ On the **Register sources (Azure SQL Database)** screen, do the following:
 
 > [!NOTE]
 > Deleting your scan does not delete your assets from previous Azure SQL Database scans.
+> This asset will no longer update the schema if re-name the column name on the source table and rescan the source table after editing the description of the schema tab of Purview.
```

Comment:
@amberz can you check this for technical accuracy? The sentence is very complex to parse. I can't quite understand actually.

```suggestion
> The asset will no longer be updated with schema changes if you rename a column on the source table and rescan the source table after editing the description in the schema tab of Purview.
```